app/client/components/dashboard/beneficiaryLiveFeed.jsx

```diff
@@ -69,13 +71,45 @@ class BeneficiaryLiveFeed extends React.Component {
                               </UserTime>
                             );
 
-                            if (transfer.transfer_type === 'PAYMENT') {
+                            let currency, fromExchangeTransfer, recipientCurrency;
+                            const transferAccountId = transfer.sender_transfer_account_id;
+                            if (transferAccountId) {
+                              const transferAccount = this.props.transferAccounts.byId[transferAccountId];
+                              currency = transferAccount && transferAccount.token && transferAccount.token.symbol;
+                            }
+                            const money = formatMoney(transfer.transfer_amount / 100, undefined, undefined, undefined, currency);
+
+
+                            if (transfer.from_exchange_to_transfer_id !== null && typeof(transfer.from_exchange_to_transfer_id) !== "undefined") {
+
+                              fromExchangeTransfer = this.props.creditTransfers.byId[transfer.from_exchange_to_transfer_id];
+                              const transferAccountId = fromExchangeTransfer.sender_transfer_account_id;
+                              if (transferAccountId) {
+                                const transferAccount = this.props.transferAccounts.byId[transferAccountId];
+                                recipientCurrency = transferAccount && transferAccount.token && transferAccount.token.symbol;
+                              }
+                              const transferToMoney = formatMoney(fromExchangeTransfer.transfer_amount / 100, undefined, undefined, undefined, recipientCurrency);
+
+                              if (transfer.transfer_type === 'EXCHANGE') {
```

Comment:
i know it's a small change, but it feels like we should put `(transfer.transfer_type === 'EXCHANGE')` on the outside at the same conditional level as `(transfer.transfer_type === 'EXCHANGE')` and put the above code inside that? just makes it easier to reason with if the same check types are all at the same level...

Reply:
Updated